index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/6d267a36ba.js" crossorigin="anonymous"></script>
+    <title>Portfolio Template</title>
```

Comment:
- [ ] Do you think that you can make this more descriptive because I am sure that this isn't just a template - it is your portfolio page to display your works, skills and abilities. Titles are very important because they have impacts on how web crawlers of Search Engines parses or understand our websites. Kindly change to something like `Your-name's Portfolio` :+1: